x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/utility_bar.tsx

```diff
@@ -125,13 +129,13 @@ export const AllRulesUtilityBar = React.memo<AllRulesUtilityBarProps>(
       <UtilityBar border>
         <UtilityBarSection>
           <UtilityBarGroup>
-            {hasBulkActions ? (
+            {hasBulkActions && pagination ? (
```

Comment:
If `hasBulkActions` is `true` and `pagination` is `undefined`, we will show exception lists count according to the logic. Is that intended?

Reply:
Yes, I agree, these are two unrelated domains, they should ahve their own components. I'll split this up.

Reply:
@xcrzx Refactored 👍